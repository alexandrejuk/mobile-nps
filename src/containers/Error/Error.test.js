import React from 'react'
import { render } from '@testing-library/react'

import ErrorContainer from '.'

describe('Error containers', () => {
  it('should render correctly', () => {
    const callbak = jest.fn()
    const { container } = render(
      <ErrorContainer goBack={callbak} />
    )
    expect(container).toMatchSnapshot()
  })
})
