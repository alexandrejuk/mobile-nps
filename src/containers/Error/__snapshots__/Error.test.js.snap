// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error containers should render correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="content"
    >
      <div
        class="image"
      >
        <img
          alt="error"
          src="error.svg"
        />
      </div>
      <h1
        class="primary large center title bold null"
      >
        OH NO...
      </h1>
      <h1
        class="secondary medium center title normal null"
      >
        Something wierg happened.
      </h1>
      <h1
        class="secondary medium center title normal null"
      >
        Keep calm and try again.
      </h1>
    </div>
    <div
      class="footer"
    >
      <button
        class="button primary null"
        type="button"
      >
        Try again
      </button>
    </div>
  </div>
</div>
`;
